New version: JustPIC v0.2.2

JuliaRegistrator opened the pull request:
- Registering package: JustPIC
- Repository: https://github.com/JuliaGeodynamics/JustPIC.jl
- Created by: @albert-de-montserrat
- Version: v0.2.2
- Commit: 01d49b87a72eee7a90e77335485633ab86df42a3
- Reviewed by: @albert-de-montserrat
- Reference: https://github.com/JuliaGeodynamics/JustPIC.jl/commit/01d49b87a72eee7a90e77335485633ab86df42a3#commitcomment-139440867
- Description: Particle-in-Cell advection
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62f62b47bb23c5fad5018e54ad24bc62bbddd9efff519f74a5af947c00a28d5205045714ac255bc691612aadaf45669f8a4ec5937b236526e8bc0bc44f065925c3181c52a67dd1fb740016f3dbf375201b08d2955275e9ffd565561a1422fdc9710b6108221d4d600da076a21c7277344e2ad39b6341c01a2993ae2dc215e193101355a671bf27d43b3ec671f769e0321f22fab6012f48627cf6ddc5d7d52d1eddbe988c55f711b28a22fa17e9c082a76 -->

albert-de-montserrat commented:
[merge approved]

albert-de-montserrat commented:
For some reason I skipped the registration of a version by mistake, and consequently the following versions didn't get auto merged (my bad I didn't realize). Is it possible to register this version without having to mess around with the old releases?

JuliaRegistrator committed:
New version: JustPIC v0.2.2

UUID: 10dc771f-8528-4cd9-9d3b-b21b2e693339
Repo: https://github.com/JuliaGeodynamics/JustPIC.jl.git
Tree: 4a4fdd69b7b0df5d31b609718ccc68416ee3c7b0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
